test/shatter/Shatter.t.sol

```diff
@@ -175,7 +176,8 @@ contract ShatterUnitTest is IERC2309Upgradeable, Test {
     }
 
     function testShatterUltra(address recipient, uint256 numShatters) public {
-        vm.assume(numShatters <= 100 && numShatters > 0);
+        vm.assume(numShatters > 1);
+        if (numShatters > 100) numShatters = numShatters % 99 + 2;
```

Comment:
Just curious what is being calculated in this if statement. For example, if numShatters = 105, then numShatters ends up = 8.

Reply:
It's a way to limit the number of shatters for testing without rejecting fuzz tests and getting failing tests due to that. https://book.getfoundry.sh/cheatcodes/assume